docs/_static/custom.css

```diff
@@ -19,3 +19,6 @@ div.rightside {
     margin-left: 125px;
 }
 
+dl.py {
```

Comment:
This is just adding some margin between each function in the API reference:

<img width="822" alt="Screenshot 2022-02-22 at 18 06 26" src="https://user-images.githubusercontent.com/5139869/155109846-af03f7e1-3062-4d36-a26b-869e3a896a6d.png">